src/drivers/localstorage.js

```diff
@@ -39,6 +39,10 @@
 
         dbInfo.keyPrefix = dbInfo.name + '/';
 
+        if (dbInfo.storeName !== self._defaultConfig.storeName) {
+            dbInfo.keyPrefix += dbInfo.storeName + '/';
+        }
```

Comment:
Test cases already exist here:

https://github.com/sebweaver/localForage/blob/issue-342-multiple-stores-on-same-db/test/test.api.js#L772

And there:

https://github.com/sebweaver/localForage/blob/issue-342-multiple-stores-on-same-db/test/test.api.js#L799

I just enriched them and they failed for localStorage too (not only IndexedDB). That's why I added the above implementation. Far simpler than for IndexedDB by the way... :-)